app/services/zendesk_intake_service.rb

```diff
@@ -63,6 +63,20 @@ def create_intake_ticket
     )
   end
 
+  def update_filing_years
```

Comment:
the intent is that this method will later be removed, right? I think maybe we should use a pattern of putting `backfill` in the name just to make it clear in the future. I assume you'll clean this up once it's done though so no worries for this commit.